authtools/views.py

```diff
@@ -61,8 +64,8 @@ class WithNextUrlMixin(object):
     success_url = None
 
     def get_next_url(self):
-        if self.redirect_field_name in self.request.REQUEST:
-            redirect_to = self.request.REQUEST[self.redirect_field_name]
+        if self.redirect_field_name in self.request.GET:
+            redirect_to = self.request.GET[self.redirect_field_name]
```

Comment:
But also, like how do you use it?

```html+django
<form method="post" action="{% url 'login' %}?next={{ next }}">
```

I guess that's OK, but it's a backwards incompatible change nonetheless